tests/helpers/mod.rs

```diff
@@ -41,6 +41,16 @@ impl TestApp {
         }
     }
 
+    pub fn override_lock(&self, contents: &str) {
+        let path = self.path().join(dmenv::DEV_LOCK_FILENAME);
+        std::fs::write(path, contents).unwrap();
+    }
+
+    pub fn override_setup_cfg(&self, contents: &str) {
+        let path = self.path().join("setup.cfg");
```

Comment:
It's strange that you used a constant above but not here and below

Reply:
Fair point. I'll do that in a later commit

Reply:
... or not. DEV_LOCK_FILENAME and PROD_LOCK_FILENAME are dmenv-specific.

`setup.cfg` and `setup.py` are not